dojo/tasks.py

```diff
@@ -291,20 +311,20 @@ def async_update_findings_from_source_issues(*args, **kwargs):
 
 @app.task(bind=True)
 def async_dupe_delete(*args, **kwargs):
-    deduplicationLogger.info("delete excess duplicates")
+    logger.info("delete excess duplicates")
     system_settings = System_Settings.objects.get()
     if system_settings.delete_dupulicates:
         dupe_max = system_settings.max_dupes
         findings = Finding.objects \
-                .filter(duplicate_list__duplicate=True) \
-                .annotate(num_dupes=Count('duplicate_list')) \
+                .filter(original_finding__duplicate=True) \
+                .annotate(num_dupes=Count('original_finding')) \
                 .filter(num_dupes__gt=dupe_max)
         for finding in findings:
-            duplicate_list = finding.duplicate_list \
+            duplicate_list = finding.original_finding \
                     .filter(duplicate=True).order_by('date')
             dupe_count = len(duplicate_list) - dupe_max
             for finding in duplicate_list:
-                deduplicationLogger.debug('deleting finding {}:{} ({}))'.format(finding.id, finding.title, finding.hash_code))
+                logger.info("Debug-Dupe: Delete Finding with id: %d", finding.id)
                 finding.delete()
                 dupe_count = dupe_count - 1
                 if dupe_count == 0:
```

Comment:
I think there is an unintended change from `deduplicationLogger` to `logger`?

Reply:
ah yes. I think this can be changed back

Reply:
should i revert this to deduplicationLogger?